ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -195,28 +193,32 @@ export const AccessRenewal = fp.flow(
   const [loading, setLoading] = useState(false);
   const [, navigateByUrl] = useNavigation();
 
+  const expirableModules = modules.filter(moduleStatus => accessRenewalModules.includes(moduleStatus.moduleName));
+
+  const completeOrBypassed = moduleName => {
+    const status = modules.find(m => m.moduleName === moduleName);
+    const wasBypassed = !!status.bypassEpochMillis;
+    return wasBypassed || !isExpiring(status.expirationEpochMillis);
```

Comment:
Existing code, but is something here perhaps misnamed? At a glance, this doesn't seem to correctly account for incomplete modules. Or - does complete here refer to "renewal complete", and we assume that all modules are complete b/c we're on the AAR page?

Reply:
yeah... this code needs some extra care.  I haven't quite wrapped my head around it, but I'm planning to come back to it soon.

Reply:
PR #6089 is intended to address this